plat/common/include/uk/plat/common/irq.h

```diff
@@ -33,8 +33,12 @@
 #ifndef __PLAT_CMN_IRQ_H__
 #define __PLAT_CMN_IRQ_H__
 
-#include <uk/plat/irq.h>
-
+/* Saving /restoring CPU IRQ flags and enabling / disabling IRQs on the
+ * CPU were previously part of the ukplat_irq API. These are not managed
+ * by the interrupt controller, so they need to be part of the lcpu
+ * low-level API. As some platforms may have their own implementation of
+ * these (linuxu, xen) keep them on a separate header.
+ */
 #if defined(__X86_64__)
```

Comment:
Normally yes. The reason I left it there is that as a reminder for whoever migrates `lcpu`, who will unlikely check the commit history. During re-arch it will go away. I'll add a `TODO` to make sure it's removed.